### DISCUSSION TITLE
正文与数学公式之间的间距小怎么办？

### DISCUSSION THREAD

**Discussion by sikouhjw** [sikouhjw/gdutthesis#34] [Show and tell]
在导言区写
```tex
\ExplSyntaxOn
\ctex_after_end_preamble:n
  {
    \setlength{\abovedisplayskip}{0.5\baselineskip - 0.5\ccwd}
    \setlength{\belowdisplayskip}{0pt}
    \setlength{\abovedisplayshortskip}{0pt}
    \setlength{\belowdisplayshortskip}{0.5\baselineskip - 0.5\ccwd}
  }
\ExplSyntaxOff
```
然后修改其中的参数即可调整公式与正文之间的间距。控制距离的命令的意义如下：
![图片](https://user-images.githubusercontent.com/49770925/148163662-a9f0d7f6-b3ee-4280-9488-4c5ef6f0fb42.png)

注意：
1. 做不到跟 word 一样的效果，如果一定要 word 的效果，请使用 word。
2. 没有两全的设置方案，将距离调大时，纵向距离小的公式会显得与正文距离大；将距离调小时，纵向距离大的公式会显得与正文距离小。

**Comment by sikouhjw** [comment]
参考[[1]]，建议使用 `align`/`gather` 环境，而不是 `equation` 环境，以此来统一上下间距。

[1]: https://www.zhihu.com/question/537764652/answer/2534890384

**Reply by FokWaijyun** [reply]
非常好

**Comment by DomOConner** [comment]
请问在”导言区“添加代码的导言区是哪里呢？
\setlength{\abovedisplayskip}{0.5\baselineskip - 0.5\ccwd}和\abovedisplayskip=12pt plus 3pt minus 9pt这两句命令，后者该如何使用呢？
现在默认情况下abovedisplayskip的情况显得公式行距很大，请问这个对于格式审查有影响吗？
latex菜鸟，不胜感激~

**Reply by note286** [reply]
在`\documentclass`和`\begin{document}`之间的位置称为导言区。

**Reply by sikouhjw** [reply]
>  \setlength{\abovedisplayskip}{0.5\baselineskip - 0.5\ccwd}和\abovedisplayskip=12pt plus 3pt minus 9pt这两句命令，后者该如何使用呢？

这两个命令的作用是一样的，只是写法不同。后者还加上了弹性距离，前者也『可以』加上。

> 现在默认情况下abovedisplayskip的情况显得公式行距很大，请问这个对于格式审查有影响吗？ 

没有，但是你可以手动调整间距使得让你看着舒服。

**Reply by LiuL22** [reply]
main.tex文件\begin{document}下面添加
%%公式间距
\ExplSyntaxOn
\ctex_after_end_preamble:n
  {
    \abovedisplayskip=10pt plus 3pt minus 8pt
    \abovedisplayshortskip=7pt plus 3pt minus 4pt
    \belowdisplayskip=10pt plus 3pt minus 8pt
    \belowdisplayshortskip=7pt plus 3pt minus 4pt
  }
\ExplSyntaxOff
感觉效果还可以

**Comment by DomOConner** [comment]
> > \setlength{\abovedisplayskip}{0.5\baselineskip - 0.5\ccwd}和\abovedisplayskip=12pt plus 3pt minus 9pt这两句命令，后者该如何使用呢？
> 
> 这两个命令的作用是一样的，只是写法不同。后者还加上了弹性距离，前者也『可以』加上。
> 
> > 现在默认情况下abovedisplayskip的情况显得公式行距很大，请问这个对于格式审查有影响吗？
> 
> 没有，但是你可以手动调整间距使得让你看着舒服。

感谢您！
我想顺便问下公式斜体加粗的问题，在其它issue中我看到您提供的解决办法是\symbfit{}或者\clist{\boldsymbol}，这两个我都尝试了没法直接编译成功，是需要在.cls或者main.tex里面做什么修改吗？
不好意思如果歪楼了我随后删掉~

**Reply by sikouhjw** [reply]
> 我想顺便问下公式斜体加粗的问题，在其它issue中我看到您提供的解决办法是\symbfit{}或者\clist{\boldsymbol}，这两个我都尝试了没法直接编译成功，是需要在.cls或者main.tex里面做什么修改吗？ 不好意思如果歪楼了我随后删掉~

`\symbfit` 编译失败吗？提供报错截图

**Comment by DomOConner** [comment]
> > 我想顺便问下公式斜体加粗的问题，在其它issue中我看到您提供的解决办法是\symbfit{}或者\clist{\boldsymbol}，这两个我都尝试了没法直接编译成功，是需要在.cls或者main.tex里面做什么修改吗？ 不好意思如果歪楼了我随后删掉~
> 
> `\symbfit` 编译失败吗？提供报错截图

就是基本的Undefined control sequence，我使用的还是1.2.7，所以里面估计有不少还没更改的内容，我想着先送盲审了，后续再用最新版慢慢调，这样可以吗？
![image](https://github.com/sikouhjw/gdutthesis/assets/45595213/18fb7519-e3b1-4190-a915-ae00b9d13cb8)

**Reply by sikouhjw** [reply]
本模板没有 1.2.7 版本，你用的是其它的模板？

**Comment by DomOConner** [comment]
> 本模板没有 1.2.7 版本，你用的是其它的模板？

是之前的XJTU-thesis，由于从开始就边写小论文边往里填东西了，所以一直没有换最新的模板，提问耽误您的时间非常抱歉~~附图是这个版本的readme.md
![image](https://github.com/sikouhjw/gdutthesis/assets/45595213/84258317-54d8-440d-b390-7ca3c205b4ed)

**Reply by sikouhjw** [reply]
本项目是 gdutthesis，跟 XJTU-thesis 无关。
